llvm/lib/Target/AIE/AIEPostPipeliner.cpp

```diff
@@ -146,7 +193,20 @@ void PostPipeliner::scheduleNode(SUnit &SU, int Cycle) {
     const int NewEarliest = Cycle + Latency;
     if (NewEarliest > Info[SNum].Earliest) {
       Info[SNum].Earliest = NewEarliest;
-      LLVM_DEBUG(dbgs() << SNum << " to " << Info[SNum].Earliest << "; ");
+      LLVM_DEBUG(dbgs() << SNum << " to " << Info[SNum].Earliest << " -; ");
+    }
+  }
+  for (auto &Dep : SU.Preds) {
```

Comment:
I think I'll need a walkthrough for that commit, there seems to be quite a lot happening on top of refactoring.